client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -49,7 +48,7 @@ function getPurchases( props ) {
 	return props.receipt.data.purchases;
 }
 
-var CheckoutThankYou = React.createClass( {
+const CheckoutThankYou = React.createClass( {
```

Comment:
It is a large component, so it might make sense as its own PR. We only have one other PR for this component open at the moment and the merge conflicts wouldn't be that bad if this PR updated `CheckoutThankYou` to use ES6.